src/app/payment/paymentMethod/CheckoutcomCustomFormFields.tsx

```diff
@@ -49,7 +49,27 @@ const Sepa: FunctionComponent<CheckoutcomAPMFormProps> = ({method}) => {
     </>);
 };
 
+const Fawry: FunctionComponent<CheckoutcomAPMFormProps> = () => {
+    return (
+        <Fragment>
+            <TextFieldForm
+                additionalClassName="form-field--customerMobile"
+                autoComplete="customerMobile"
```

Comment:
just a usability tip, I wonder if for the autocomplete value we want to match with the autocomplete in the shipping/billing form, so it automatically suggests the values entered there.